editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -9,11 +9,13 @@ import { Store, useStore } from "./store";
 export type AnalyticsType = "init" | "resume";
 type AnalyticsLogDirection = AnalyticsType | "forwards" | "backwards";
 
+export type HelpClickMetadata = Record<string, string>;
```

Comment:
This wasn't exactly what I had wanted here initially - I thought something like this would be a bit clearer - 

```ts
export interface HelpClickMetadata {
  fn: string;
  name: string;
}
```

And we could then keep an array of these in the `metadata` column.

The native `_append` in Hasura is a little limited, and I couldn't simply do this without either - 
 - always overwriting the previous value
 - tracking `HelpClickMetadata` in state locally and modifying this object to overwrite the previous value
 
I've gone for this as it's the simplest option for now.

Reply:
Docs: https://hasura.io/blog/postgres-json-and-jsonb-type-support-on-graphql-41f586e47536/